src/story.py

```diff
@@ -0,0 +1,34 @@
+""" Story class file """
+
+class Story(object):
+    """ Describe a story.
+    States, items, traits, class
+    """
+
+    @staticmethod
+    def load_from_json(json_content):
+        """Return a Story object load from the json"""
+        pass
```

Comment:
but why

Reply:
why what ? the `@staticmethod` ?

Reply:
yes. why not just have a free function? having it as part of the class is pointless

Reply:
In my opinion, this is exaclty why static method exists... But maybe I'm wrong in python ?

Reply:
well your opinion is wrong, dont do this

Reply:
>.> no more explication ?

Reply:
youre putting a function that is not dependant on the class itself inside the class. its just messy code. its cleaner to keep the function at top level without putting it in a class just for the sake of it.

a staticmethod doesnt have access to the attirbutes or the instance itself so why not just keep it outside the class and save some boilerplate and needless coupling.

Reply:
also might i add, using staticmethods like this is usually looked down upon in the python community because generally its unneccessary

Reply:
also to note: if youre doing this to help namespace the function, python already has namespaces in the form of modules.

Reply:
Well, for me this function act as a constructor, in C++ I would make as static. But if it's a bad practice in python, then...

Reply:
if you want to make a constructor in python, you would usually do that in __init__ but technically the actual constructor method is __new__ i believe